modules/task/task.go

```diff
@@ -56,6 +46,12 @@ func GlobalBackend() Backend {
 	return _globalBackend
 }
 
+func globalBackendStatsClient() *statsClient {
```

Comment:
To factor this out when we get rid of the globals in task altogether hopefully, but getting stats down to just this is the start